src/main/scala/edu/berkeley/ce/sparkrocks/LinearProgram.scala

```diff
@@ -21,9 +26,10 @@ object LinearProgram {
   * @constructor Initialize a new linear program.
   * @param numVars The number of variables in the linear program.
   */
-class LinearProgram(val numVars: Int) {
-  val solver = LpSolve.makeLp(0, numVars)
-  solver.setVerbose(LpSolve.IMPORTANT)
+class LinearProgram(numVars: Int) {
```

Comment:
Cool, that makes sense. Hadn't really thought about this at all until I saw the `val` in there; just assumed it never was necessary...